src/gc/unix/gcenv.unix.cpp

```diff
@@ -530,6 +532,9 @@ uint32_t GCToOSInterface::GetCurrentProcessCpuCount()
     if (count == 0 || count > 64)
         count = 64;
 
+    if (cgroup.GetCpuLimit(&cgroupCpuCount)){
```

Comment:
Yes indeed sorry about that. I wonder how it could have passed all the checks.
I will update the code on Monday. Thank you!